### DISCUSSION TITLE
Backlinks inside footnotes are not highlighted in PDFs

### DISCUSSION THREAD

**Discussion by piisamirontta** [RyotaUshio/obsidian-pdf-plus#287] [Q&A]
I recently switched to using inline footnotes for citing references, including the PDF++ backlink. My footnotes look like this:
`^[@ref: [[link-to-pdf]]]`. Since the switch, the backlinked text no longer appears highlighted in the PDF viewer. Old backlinks are still highlighted, despite having been edited to be inside a footnote. I'm not sure if this is a bug or intended behavior. Is there some option I could tinker with to get PDF++ to search for backlinks inside footnotes?

Attached is my PDF++ data file:
[data.json](https://github.com/user-attachments/files/17450790/data.json)

**Comment by RyotaUshio** [comment]
This is in fact a result of a bug in the Obsidian core rather than PDF++. The current version of Obsidian cannot properly index links inside **inline** footnotes, causing many problems including this one. There is a bug report on the forum: https://forum.obsidian.md/t/backlinks-not-working-on-notes-referenced-via-inline-footnote/89269

They say it will be fixed in v.1.8.

Unfortunately, it's not what a plugin can address - only the Obsidian team can fix this. For the time being, you can use normal (non-inline) footnotes instead. Links in normal footnotes work flawlessly.
You will find the [Footnote Shortcut](https://github.com/akaalias/obsidian-footnotes) plugin useful. There's also Obsidian's native autocompletion for footnotes (type `[^`).
